src/providers/components/loaders/ServerListProvider.ts

```diff
@@ -0,0 +1,10 @@
+export default class ServerListProvider {
```

Comment:
These usually throw errors if they're accessed and not defined. I think it'd be best to follow that format.

They're usually for explicit on-screen warnings so that they don't fail silently for the user, and so you don't need to run checks to see if they're defined in other areas. In theory we should always have a provider set.